cloudaux/orchestration/aws/events.py

```diff
@@ -0,0 +1,77 @@
+from cloudaux.aws.events import describe_rule
+from cloudaux.aws.events import list_targets_by_rule
+from cloudaux.decorators import modify_output
+from flagpole import FlagRegistry, Flags
+
+from botocore.exceptions import ClientError
+import logging
+import json
+
+
+logger = logging.getLogger('cloudaux')
+
+
+registry = FlagRegistry()
+FLAGS = Flags('BASE', 'DESCRIBE', 'TARGETS')
+
+
+@registry.register(flag=FLAGS.TARGETS, key='targets')
+def list_targets(rule_name, **conn):
+    return list_targets_by_rule(Rule=rule_name, **conn)
+
+
+@registry.register(flag=FLAGS.DESCRIBE)
+def get_rule_description(rule_name, **conn):
+    rule = describe_rule(Name=rule_name, **conn)
+
+    rule_detail = None
+
+    if rule.get('ScheduleExpression', None):
+        rule_detail = rule['ScheduleExpression']
+    else:
+        rule_detail = rule['EventPattern']
+
+    return {
+        'description': rule['Description'],
+        'state': rule['State'],
+        'rule': rule_detail
+    }
+
+
+@registry.register(flag=FLAGS.BASE)
+def get_base(rule_name, **conn):
```

Comment:
Seems odd that `get_base` would work even when the underlying event rule does not exist.

I think the other cloudaux orchestrators typically call the describe function from within the `get_base` method, but only if the base_fields are not already set:

https://github.com/Netflix-Skunkworks/cloudaux/blob/master/cloudaux/orchestration/aws/elbv2.py#L57